test/suite/lsp/server/diagnostics/gql/over-sized-record.test.ts

```diff
@@ -16,22 +16,23 @@ import {
     OversizedRecord,
     OVER_SIZED_FIELD_MESSAGE,
     OVER_SIZED_RECORD_MESSAGE
-} from '../../../src/diagnostic/gql/over-sized-record';
-import { OrgUtils } from '../../../src/utils/orgUtils';
+} from '../../../../../../src/lsp/server/diagnostics/gql/over-sized-record';
+import { ObjectInfoUtils } from '../../../../../../src/lsp/server/utils/objectInfoUtils';
 
-import { ObjectInfoRepresentation } from '../../../src/types';
+import { ObjectInfoRepresentation } from '../../../../../../src/lsp/server/types';
 
 suite(
     'GraphQL Diagnostics Test Suite - Server - Oversized GraphQL Field',
     () => {
+        let sandbox: sinon.SinonSandbox;
         let oversizedRecordProducer = new OversizedRecord();
+        const book = JSON.parse(readFileSync('test/suite/lsp/server/testFixture/objectInfos/Book__c.json', 'utf-8'));
```

Comment:
`resolveJsonModule` is no longer defined in `tsconfig`. So I'm reading json using `fs` instead.

Reply:
Actually resurrected `resolveJsonModule` and removed reading json using `fs`. I changed how the code was `import`ing JSONs to make them work.